src/actinia_statistic_plugin/response_models.py

```diff
@@ -588,3 +586,153 @@ class RasterAreaUnivarStatsResponseModel(ProcessingResponseModel):
         },
         "user_id": "admin",
     }
+
+
+class RasterSamplingResponseModel(ProcessingResponseModel):
+    """Response schema for a raster map sampling result.
+
+    This schema is a derivative of the ProcessingResponseModel that defines a different
+    *process_results* schema.
+    """
+
+    type = "object"
+    properties = deepcopy(ProcessingResponseModel.properties)
+    properties["process_results"] = {}
+    properties["process_results"]["type"] = "array"
+    properties["process_results"]["items"] = {
+        "type": "array",
+        "items": {"type": "string", "minItems": 3},
+    }
+    required = deepcopy(ProcessingResponseModel.required)
+    example = {
+        "accept_datetime": "2022-03-08 14:44:04.264502",
+        "accept_timestamp": 1646750644.2644997,
+        "api_info": {
+            "endpoint": "syncephemeralrastersamplingresource",
+            "method": "POST",
+            "path": "/api/v3/locations/nc_spm_08/mapsets/PERMANENT/raster_layers/landuse96_28m/sampling_sync",
+            "request_url": "http://localhost{URL_PREFIX}/locations/nc_spm_08/mapsets/PERMANENT/raster_layers/landuse96_28m/sampling_sync",
+        },
+        "datetime": "2022-03-08 14:44:04.656022",
+        "http_code": 200,
+        "message": "Processing successfully finished",
+        "process_chain_list": [
+            {
```

Comment:
see other comment

Reply:
Changed to new PC style in 214b495018bc9cfb928779892b710c1e0cd791e9